2018/2018.md

```diff
@@ -0,0 +1,5 @@
+## 2018
+
+| Conference | Date | Venue | Description |
+|------------|------|-------|-------------|
+| [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google and many contributors from the open source community. |
```

Comment:
There are more conferences listed out for 2018 in the README. Please add all of them.